### PR TITLE
STORM-1664: Allow Java users to start a local cluster with a Nimbus T…

### PR DESCRIPTION
**Pull request by srdo**
…hrift server.

### CODE REVIEW

**Comment by knusbaum** [comment]
Is there a reason for wanting this? I see you've made the change but no code is using it.

**Comment by srdo** [comment]
Hi, yes it's useful for running https://github.com/roshannaik/storm-benchmark-sol in local mode, since that code uses NimbusClient for pulling throughput metrics.

File: storm-core/src/clj/org/apache/storm/testing4j.clj [hunk 1]
```diff
@@ -79,10 +79,12 @@
   [cluster-type mkClusterParam code]
   `(let [supervisors# (or (.getSupervisors ~mkClusterParam) 2)
          ports-per-supervisor# (or (.getPortsPerSupervisor ~mkClusterParam) 3)
-         daemon-conf# (or (.getDaemonConf ~mkClusterParam) {})]
+         daemon-conf# (or (.getDaemonConf ~mkClusterParam) {})
+         nimbus-daemon# (or (.isNimbusDaemon ~mkClusterParam) false)]
```
Comment (knusbaum) [inline storm-core/src/clj/org/apache/storm/testing4j.clj]: The `or` here is unnecessary.

Comment (srdo) [inline storm-core/src/clj/org/apache/storm/testing4j.clj]: Right, fixed.

**Comment by knusbaum** [comment]
+1 pending Travis

**Comment by srdo** [comment]
@knusbaum Mind merging if you're still +1 on this change?

**Comment by HeartSaVioR** [comment]
+1

**Pull request merged**